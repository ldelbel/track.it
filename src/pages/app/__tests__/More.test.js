import React from 'react';
import renderer from 'react-test-renderer';
import More from '../more';

describe('more component',() => {
    it('renders correctly', () => {
        const more = renderer.create(<More  />).toJSON();
        expect(more).toMatchSnapshot();
    });
})