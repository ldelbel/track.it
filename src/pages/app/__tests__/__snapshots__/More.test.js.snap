// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`more component renders correctly 1`] = `
<main
  className="bg"
>
  <div
    className="container"
  >
    <button
      className="button"
      onClick={[Function]}
      type="button"
    >
      Logout
    </button>
  </div>
</main>
`;
